dotCMS/src/main/java/com/dotmarketing/db/HibernateUtil.java

```diff
@@ -889,6 +889,45 @@ public static void startTransaction()  throws DotHibernateException{
 		}
 	}
 
+	/**
+	 * This just commit does not close anything.
+	 * @throws DotHibernateException
+	 */
+	public static void onlyCommitTransaction()  throws DotHibernateException{
+
+		try{
+			// if there is nothing to close
+			if (sessionHolder.get() == null){
+				return;
+			}
+			Session session = getSession();
+
+			if (session != null) {
+				session.flush();
+				if (!session.connection().getAutoCommit()) {
+					Logger.debug(HibernateUtil.class, "Closing session. Commiting changes!");
+					session.connection().commit();
+					session.connection().setAutoCommit(true);
+					if(commitListeners.get().size()>0) {
+						finalizeCommitListeners();
+					}
+				}
+			}
+		}catch (Exception e) {
+			Logger.error(HibernateUtil.class, e.getMessage(), e);
+			throw new DotHibernateException("Unable to close Hibernate Session ", e);
+		}
+		finally {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this "finally" on the same line that the previous closing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)